crates/ty_server/src/capabilities.rs

```diff
@@ -194,3 +220,13 @@ impl ResolvedClientCapabilities {
         flags
     }
 }
+
+/// Creates the default [`DiagnosticOptions`] for the server.
```

Comment:
This feels a bit out of place. The function doesn't contain any code related to capabilities